### DISCUSSION TITLE
Slider breaks even though inputs seem to be correct.

### DISCUSSION THREAD

**Discussion by Inconsist** [ppy/osu#22336] [Q&A]
I hope this isnt a know issue or I just dont know how to play. The inputoverlay is just enlarged the timing is correct.

https://user-images.githubusercontent.com/87947227/213895910-8826eb4e-4387-4474-b977-4f6453d947ff.mp4

**Comment by bdach** [comment]
Can you upload the replay?

**Reply by Inconsist** [reply]
[Halflight playing Leah Kate - Life Sux (Sped Up Ver.) (Log Off Now) [Mommy Sucks] (2023-01-22_01-50).zip](https://github.com/ppy/osu/files/10475630/Halflight.playing.Leah.Kate.-.Life.Sux.Sped.Up.Ver.Log.Off.Now.Mommy.Sucks.2023-01-22_01-50.zip)



![image](https://user-images.githubusercontent.com/87947227/213942894-e9a2c80b-f2dd-4e28-a099-071d881c35b8.png)

**Reply by integerrr** [reply]
Judging from the replay B2 was used to hit the sliderhead, which was then subsequently released and causing the slider break, looks like intended behaviour here

**Reply by Inconsist** [reply]
Yes but b1 was always held down, and since b2 was released immediately why would it stay hit til the reverse arrow? You can actually alternate tap whilst on a slider without it breaking if you do it right so I dont see why this doesnt work here.

**Reply by integerrr** [reply]
B1 wasn't used to hit the slider head so the slider doesn't track whether B1 is held or not.

The reason why it only broke combo until hitting a slider repeat and not earlier is the same reason why hitting a slider and then releasing the key immediately only breaks combo at the slider ticks and not earlier.

**Reply by Inconsist** [reply]
I can see why you might want to impliment it that way, so people dont hold down one key and then use another to hit the sliderheads, but then again you can switch the keys while tracking the slider. Something about this feels off, at least to me.

**Reply by integerrr** [reply]
Switching keys is different to having said keys held down before the slider is hit originally, by switching keys you are providing additional key down events, which B1 in the original replay does not do
